pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -483,10 +483,12 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				status := nodePortFilter.Filter(context.TODO(), state, nil, nodeInfo)
 				nodePortStatus := api.ConvertPredicateStatus(status)
 				if nodePortStatus.Code != api.Success {
+					// TODO: Currently, preemption is not supported when NodePort filtering fails.
+					// Once supported, the logic here should be removed.
+					// See https://github.com/volcano-sh/volcano/issues/3845
+					nodePortStatus.Code = api.UnschedulableAndUnresolvable
```

Comment:
Why does a node port not support preemption?

Reply:
Similar to inter-pod affinity, passing resource comparison during preemption does not guarantee that the pod occupying the node’s port will be preempted.